stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/shared/media/ChatInfoSharedMediaFragment.kt

```diff
@@ -119,13 +122,20 @@ class ChatInfoSharedMediaFragment : Fragment() {
     }
 
     private fun showEmptyState() {
+        setActionBarElevation(ACTION_BAR_ELEVATION)
         binding.progressBar.isVisible = false
         binding.mediaRecyclerView.isVisible = false
         binding.dateView.isVisible = false
         binding.emptyStateView.isVisible = true
         scrollListener.disablePagination()
     }
 
+    private fun setActionBarElevation(elevation: Float) {
+        (requireActivity() as AppCompatActivity).run {
+            supportActionBar?.elevation = elevation
```

Comment:
Isn't this the `Toolbar` (`@+id/toolbar`) in our own Fragment? Do we need to go through the Activity for this?

Reply:
Good catch, thanks!